src/internal/connector/onedrive/restore.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/alcionai/corso/src/internal/connector/graph"
 	"github.com/alcionai/corso/src/internal/connector/support"
 	"github.com/alcionai/corso/src/internal/data"
+	D "github.com/alcionai/corso/src/internal/diagnostics"
```

Comment:
Is there precedent for using an import alias like `D`? I haven't seen this before.

Reply:
Something I've personally done before when a common library has a longer name.  No strong feelings about it.